### PR TITLE
minor name changes

### PR DESCRIPTION
**Pull request by kroos47**

### CODE REVIEW

File: client/src/lib/assets/FAQ.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ To request funds, simply enter your <NETWORK-NAME> wallet address, fill the capt
...
+You can request tokens every 24h
 
 ## What is a testnet token?
 
 Testnet tokens are a test currency that allows you to test the network before going live. Testnet tokens can be used in place of real tokens on testnets like <NETWORK-NAME>.
 
 ## What if I run into any other issues, or have questions?
 
-You can [report an issue](https://github.com/availproject/polkadot-testnet-faucet/issues) in our repository or ask in our discord channel!
+You can [report an issue](https://github.com/availproject/polkadot-testnet-faucet/issues) in our repository or ask in our [discord](https://discord.gg/kbUNx6vB)!
```
Comment (moraesjeremias) [inline client/src/lib/assets/FAQ.md:19]: We need to enable issues on this repo. Requests are forwarded to PR section.

File: client/.gitignore [hunk 2]
```diff
@@ -31,3 +31,4 @@ vite.config.ts.timestamp-*
 #####################
 /.env
 !.env.example
+*.env
```
Comment (moraesjeremias) [inline client/.gitignore:34]: EOF

**Review by moraesjeremias — APPROVED** [review]
LGTM!
Small fixes that can be address later. Let's merge it!

File: client/src/lib/assets/FAQ.md (hunk 1 above)
Comment (mordamax) [reply client/src/lib/assets/FAQ.md:19]: @moraesjeremias @kroos47 could you please actually enable those? as paritytech started getting reports to your project

**Pull request merged**